### ISSUE TITLE
IS THIS PROJECT UNMAINTAINED?

### ISSUE THREAD

**Issue by kaminskypavel** [mobnetic/BitcoinChecker#831]
we love the project, please find a maintainer!

**Comment by xziy** [comment]
https://github.com/m42cx/BitcoinChecker/releases

**Comment by kaminskypavel** [comment]
@xziy is this a decompiled version of the apk? how do you connect the data module? or does it resides in the same repo?

**Comment by xziy** [comment]
Yes this decompile, we start to clone this program. Just wait news

**Comment by MaciejCiemiega** [comment]
Hello,
due to new Google Play policy all apps should target Android Oreo starting from November 2018, I would not be able to publish updates after that date based on the current code. Since the application's code is very old (was mostly unchanged for last few years) and it requires a some changes in order to target Oreo. Also the changes would be quite tricky because Bitcoin Checker offers a lot of customization of notifications, which is very restricted if the app targets Android Oreo.

In order to unblock this project, I plan to:

1. Take a look at PRs in following days, merge them if possible and release.
2. Start migrating to Oreo. I would probably need some help to decide how to handle it in the best way because I want to remove as little customization features as possible (otherwise users would be upset)
3. Try to rewrite some code and make it open sourced as well, alongside the dataModule So people can contribute not only to dataModule but to the app itself as well.
4. Find some people to help to maintain the project (I will create separate Issue for that later).

What do you think about that?

**Comment by kaminskypavel** [comment]
@MaciejCiemiega thanks for taking the time to update. I love the project and would love to see it thrives rather than fade away. 
Do let me (and other developers) know how can we help (both with data module and the apk)

**Comment by xziy** [comment]
@MaciejCiemiega, Need look source, you can share source or just send me zip for to study.

**Comment by MaciejCiemiega** [comment]
I wouldn't call it "brand new" and "shiny" but yeah, I will release update this week:)
